core/actions/googleDriveImport/formatDriveData.ts

```diff
@@ -116,12 +116,12 @@ export const formatDriveData = async (dataFromDrive: DriveData): Promise<Formatt
 						values: {
 							"arcadia:anchor":
 								index === 0 && discussion.anchors.length
-									? discussion.anchors[0]
+									? JSON.stringify(discussion.anchors[0])
 									: undefined,
 							"arcadia:content": comment.text,
 							"arcadia:publication-date": comment.createdAt,
 							"arcadia:full-name": comment.author.fullName,
-							"arcadia:orcid": comment.author.orcid,
+							"arcadia:orcid": `https://orcid.org/${comment.author.orcid}`,
```

Comment:
Current use pages on legacy just render the orcid value - (and construct the URL for the href). Is there a reason to store the whole URL, as opposed to just the ORCID? Somehow feels cleaner to me to construct the URL when we need, than to try and parse out just the ID value when we need it.

Reply:
Yes. Storing them as just the ID part causes a bunch of problems on Legacy because ORCIDs are technically fully qualified URIs — the https:// is part of the ID — and people are used to pasting in the full URL on every other system. All the validators are built assuming this. Etc. So on Legacy we're parsing in and parsing out all the time anyway.